### DISCUSSION TITLE
[Feature] ios app support select photo and video upload

### DISCUSSION THREAD

**Discussion by KleinerSource** [immich-app/immich#10841] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

ios app support select photo and video upload

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Please put some more effort into requests, it's really hard to tell what you're asking for ;) That said, I think this is almost certainly a duplicate.

**Reply by KleinerSource** [reply]
I hope the ios app supports manual selection of photos and videos to upload instead of just synchronizing photo albums
